env-var-docs/generate_markdown.py

```diff
@@ -0,0 +1,155 @@
+"""Generates markdown from an environment variable documentation file and write it to a github repository.
+
+Requires the following variables to be present in the environment:
+    ENV_VAR_DOCS_PATH: the path to env-var-docs.json.
+
+If LOCAL_OUTPUT is set, markdown will be written to stdout instead of GitHub.
+
+If LOCAL_OUTPUT is not set, the following variables must be set:
+    RELEASE_VERSION: CiviForm version being released. 
+    GITHUB_ACCESS_TOKEN: personal access token to call github API with.
+    TARGET_REPO: the target github repository in 'owner/repo-name' format.
+    TARGET_PATH: the relative path within TARGET_REPO to write the file to.
+"""
+
+import dataclasses
+import env_var_docs.parser
+import env_var_docs.errors_formatter
+import github
+import os
+import sys
+import typing
+
+
+def errorexit(msg):
```

Comment:
**nitpick:** 
```suggestion
def error_exit(msg):
```

Reply:
For consistency with the other scripts I will leave as errorexit